src/components/Emoji.tsx

```diff
@@ -1,7 +1,14 @@
 import React from "react"
 import { Twemoji } from "react-emoji-render"
 import styled from "styled-components"
-import { margin } from "styled-system"
+import { margin, MarginProps } from "styled-system"
+
+interface IProp extends MarginProps {
+  size: number
+  text: string
+}
+
+// const StyledTweomoji: React.FC<Twemoji>
 
 const StyledEmoji = styled(Twemoji)`
   & > img {
```

Comment:
Twemoji's prop has no `.size` property. In order to pass it in, it seems like it needs to be a way to override its FC signature